code/modules/events/vent_clog.dm

```diff
@@ -130,6 +133,9 @@
 			foam.start()
 		CHECK_TICK
 
+/datum/round_event/vent_clog/female/announce()
+	priority_announce("The scrubbers network is experiencing a backpressure surge. Some ejection of contents may occur.", "Atmospherics alert")

```

Comment:
Idem

Reply:
It was originally trilby code :')

Reply:
Thing is it doesn't proc succesfully without this. It makes it spawn smoke instead of foam, using the old scrubber backfire pre-goof foam. This is some Magic Byond Coding to me.

Reply:
All this proc does is announcing what's going on to the crew, It should not be affecting anything else.
But if what you say is right, then something would be really, really off, specially since the only smokeful vent clogging event nowadays is the plasma decontamination one.

Reply:
Yeah man I have no fucking idea.